Sources/TensorFlow/Operators/Basic.swift

```diff
@@ -1239,3 +1239,25 @@ extension Tensor {
         return areAxesInRange(axes.scalars)
     }
 }
+
+/// Checks to ensure elements within an array are distinct.
```

Comment:
First of all, thank you very much for the detailed review. I appreciate it. Since this functionality isn't part of the original motivation for this PR, I've removed it, and will send it along (having (hopefully) learned from your comments) in a subsequent PR.